docs: add vue school links

danielkellyio opened the pull request:
## Description
These ads are as discussed with @hootlex.

## Motivation and Context
Provide educational and promotional links to Vue School.

## How Has This Been Tested?
Add links in local environment and visually checked.

## Types of changes
<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist:
<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and backwards compatible changes and `next` for non-backwards compatible changes).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

DRoet commented on packages/docs/src/pages/en/components/data-tables.md:
```diff
@@ -13,7 +13,9 @@ related:
 
-<promoted-ad type="theme" />
+<vue-school-link href="https://vueschool.io/?friend=vuetify">
+Up you Vue.js game and support Vuetify by subscribing to Vue School
```
```suggestion
Up your Vue.js game and support Vuetify by subscribing to Vue School
```

danielkellyio commented:
updated `vue-school-ad` to `vue-school-link` to prevent issues with ad blockers and updated wording of cta's to sound more actionable. 

My apologies for not having these things in place when creating the initial PR

danielkellyio commented:
this should be good to go based on latest feedback. Thanks!
